src/dotty/tools/dotc/core/Symbols.scala

```diff
@@ -398,10 +398,10 @@ object Symbols {
 
     /** Subclass tests and casts */
     final def isTerm(implicit ctx: Context): Boolean =
-      (if(isDefinedInCurrentRun) lastDenot else denot).isTerm
+      (if (defRunId == ctx.runId) lastDenot else denot).isTerm
```

Comment:
Because it's used elsewhere to mean: In a compilation unit that's currently being compiled.